eng/sdk_files/Emscripten.Sdk.props

```diff
@@ -1,6 +1,12 @@
 <Project>
-    <PropertyGroup>
-        <EmscriptenSdkToolsPath>$(MSBuildThisFileDirectory)..\tools\</EmscriptenSdkToolsPath>
-        <EmSdkUpstreamBinPath>$(EmscriptenSdkToolsPath)bin\</EmSdkUpstreamBinPath>
-    </PropertyGroup>
+  <PropertyGroup>
+    <EmscriptenSdkToolsPath>$(MSBuildThisFileDirectory)..\tools\</EmscriptenSdkToolsPath>
+    <EmscriptenUpstreamBinPath>$(EmscriptenSdkToolsPath)bin\</EmscriptenUpstreamBinPath>
+    <EmscriptenUpstreamEmscriptenPath>$(EmscriptenSdkToolsPath)emscripten\</EmscriptenUpstreamEmscriptenPath>
```

Comment:
does upstream need to get added to the path?

Reply:
Yeah, `upstream/bin` has `wasm-ld`, and `upstream/emscripten` has `emcc`.